tests/functional/spec/features/supplementary_data/supplementary_data_non_list_items.spec.js

```diff
@@ -0,0 +1,130 @@
+import HubPage from "../../../base_pages/hub.page";
+import IntroductionBlockPage from "../../../generated_pages/supplementary_data_all_variations/introduction-block.page.js";
+import Section1Page from "../../../generated_pages/supplementary_data_all_variations/section-1-summary.page.js";
+import EmailBlockPage from "../../../generated_pages/supplementary_data_all_variations/email-block.page.js";
+import NewEmailPage from "../../../generated_pages/supplementary_data_all_variations/new-email.page.js";
+import SalesBreakdownBlockPage from "../../../generated_pages/supplementary_data_all_variations/sales-breakdown-block.page.js";
+import CalculatedSummarySalesPage from "../../../generated_pages/supplementary_data_all_variations/calculated-summary-sales.page.js";
+import Section1InterstitialPage from "../../../generated_pages/supplementary_data_all_variations/section-1-interstitial.page.js";
+import TradingPage from "../../../generated_pages/supplementary_data_all_variations/trading.page";
+import ListCollectorPage from "../../../generated_pages/supplementary_data_all_variations/list-collector.page";
+import AnyEmployeesPage from "../../../generated_pages/supplementary_data_all_variations/any-employees.page";
+import Section2Page from "../../../generated_pages/supplementary_data_all_variations/section-2-summary.page.js";
+import AddEmployeePage from "../../../generated_pages/supplementary_data_all_variations/list-collector-add.page.js";
+import Section3Page from "../../../generated_pages/supplementary_data_all_variations/section-3-summary.page.js";
+import LengthOfEmploymentPage from "../../../generated_pages/supplementary_data_all_variations/length-of-employment.page.js";
+import SupermarketTransportPage from "../../../generated_pages/new_calculated_summary_repeating_and_static_answers/supermarket-transport.page";
+
+describe("Using supplementary data non list items", () => {
```

Comment:
Can we enable the section summary and it ensure that it renders fine, worth doing view submitted response also.

Reply:
Done 👍 - I've noticed that, the section context is build via:
`_get_refactored_groups` turning `group-3` into `group-3-0`
and then the summary context builds all the groups, which creates duplicates of `group-3-0` and then `set_unique_group_ids` sets this to `group-3-0` and `group-3-0-1`

This probably isn't quite what is wanted on the summary page, but it may be preferable for this to be fixed in a follow up ticket (? ), it just means that for my functional tests, the last two selectors use an odd group content index

Reply:
Thanks, I see that it is done for view submitted but can you add it for section summary also?. 

> I've noticed that, the section context is built via:
_get_refactored_groups turning group-3 into group-3-0
and then the summary context builds all the groups, which creates duplicates of group-3-0 and then set_unique_group_ids sets this to group-3-0 and group-3-0-1

> This probably isn't quite what is wanted on the summary page, but it may be preferable for this to be fixed in a follow up ticket (? ), it just means that for my functional tests, the last two selectors use an odd group content index

Agreed. It complicates the selectors however let's not spend that on that now. That is some tech debt we can progress in the near future.